.github/workflows/build.yml

```diff
@@ -11,11 +20,12 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: '3.8'
+          python-version: '3.x'
```

Comment:
as to why 3.x, quoting docs:

> For most operating systems, Python is already present, but for Windows you may need to install it. Starting October 2019, Python 3.x is supported in all build tools we use for ESPEasy.

at least this will catch any possible issues with future python versions.

although, pio-core wants 3.6+ (per https://docs.platformio.org/en/latest/core/installation.html#system-requirements), but theirs CI only tests 3.6...3.9

Reply:
Ah, I thought it was an issue with the docs... Maybe read it wrong.
As far as I know, I am using Python 3.8 on Windows. No idea what versions the Linux machines run, but that doesn't really matter as it is always run in a virtual env, so you can pick any Python version you like.

Reply:
nonetheless, it builds now and I don't see any issues with build and the resulting documentation html
(also as with releases, attached to the workflow run as artifact)